### ISSUE TITLE
Color maps for Plots -> Allow for different colormaps, change the default

### ISSUE THREAD

**Issue by jnaiman** [data-carpentry-for-agriculture/trial-lesson#62]
Right now it's hard to see the yellow on the screen.  In `map_points` there is an option for changing the colormap, this needs to be added to the `map_poly`.

Also, we might want to modify the default colormap so its easier to see on the screen

**Comment by dlstrong** [comment]
Yeah, yellow on white is rough! Black? Brown? Dark blue? (Is there a way to change that in either the load-and-test script or the functions script so that we don't have to retype it every time?)
